packages/keycodes/src/index.js

```diff
@@ -353,7 +353,12 @@ export const isKeyboardEvent = mapValues( modifiers, ( getModifiers ) => {
 		const mods = getModifiers( _isApple );
 		const eventMods = getEventModifiers( event );
 
-		if ( xor( mods, eventMods ).length ) {
+		const modsDiff = mods.filter( ( mod ) => ! eventMods.includes( mod ) );
+		const eventModsDiff = eventMods.filter(
+			( mod ) => ! mods.includes( mod )
+		);
+
+		if ( new Set( [ ...modsDiff, ...eventModsDiff ] ).size > 0 ) {
```

Comment:
Wouldn't it be better to simply test for `modsDiff.length > 0 || eventModsDiff.length > 0`?

Reply:
Addressed as part of https://github.com/WordPress/gutenberg/pull/43389/commits/6df8ac5a7f15c943c09614d7a13e1ac0d7471729